services/movie_session.py

```diff
@@ -0,0 +1,54 @@
+from django.db.models import QuerySet
+
+from db.models import MovieSession, Movie, CinemaHall
+
+
+def get_movies_sessions(session_date: str = None) -> QuerySet:
+    queryset = MovieSession.objects.all()
+    if session_date:
+        queryset = queryset.filter(show_time__date=session_date)
+    return queryset
+
+
+def create_movie_session(
+        movie_show_time: str,
+        movie_id: int,
+        cinema_hall_id: int
+) -> QuerySet:
```

Comment:
The return type of the `create_movie_session` function is incorrectly specified as `QuerySet`. It should be `MovieSession` since the function returns a single instance of `MovieSession`, not a queryset.